apps/coscrad-frontend/src/utils/generic-components/presenters/single-property-presenter.tsx

```diff
@@ -15,8 +15,9 @@ export const SinglePropertyPresenter = <T,>({
         <>
             <Typography component={'span'} sx={{ fontWeight: 'bold' }}>
                 {display}:
-            </Typography>{' '}
-            {value}
+            </Typography>
+            &nbsp;
+            <Typography component={'span'}>{value as string}</Typography>
```

Comment:
added typography because the styling of the popup is pretty chaotic with multiple over-lapping css declarations from leaflet interferring.

Reply:
Ah- this is a downside of using leaflet I suppose.